packages/universal/collections/README.md

```diff
@@ -0,0 +1,120 @@
+# Starbeam Collections
+
+Starbeam collections are reactive implementations of JavaScript built-in
+collections.
+
+- [Object]
+- [Array]
+- [Map]
+- [Set]
+- [WeakMap]
+- [WeakSet]
+
+Starbeam collections behave identically to the JavaScript built-ins that they
+implement.
+
+> 📢 Starbeam Collections do not support prototype mutation. This is the only known
```

Comment:
does `instanceof` still work? (e.g.: `reactive.map() instanceof Array`) I think tracked-built-ins made it work, I forgot about starbeam -- may be worth calling out somewhere?

Reply:
I intentionally didn't make that work. I would be open to it, but it creates a lot of extra complexity and isn't really what the use-case for these features is about.

> I think it would actually be pretty easy to make `instanceof` work, but in tracked-builtins, that also meant that we tried to make the constructor compatible. I really don't want to do that, because it makes it very difficult to pass extra reactive parameters like `description` and `equals`, and afaict there is basically no use-case where some external code takes "any constructor for `Map`" as a parameter that we would need to be compatible with.

My current rule is:

> `ReactiveMap<K, V>` should have 1:1 correspondence with the instance type, but there is no correspondence between `typeof Map` and `reactive.Map().constructor`.

Reply:
(I'm open to a discussion about changing this, but that's how it works now, and I think that instance-side correspondence is what people want/need).